src/components/AdvertsTableRow/AdvertsTableRow.tsx

```diff
@@ -263,7 +270,7 @@ const AdvertsTableRow = memo((props: TAdvertsTableRowRenderer) => {
                                     className='lg:min-w-[7.5rem]'
                                     disabled={isAdvertiserBarred || !isScheduleAvailable}
                                     onClick={() => {
-                                        if (!isAdvertiser && !isPoiPoaVerified) {
+                                        if (!isAdvertiser && (!isPoiPoaVerified || shouldShowVerification)) {
```

Comment:
suggestion:
we can turn this to a new hook like `useIsAdvertiserVerified()`. because I saw this check in other places as well so it can be reusable.

Reply:
true ill do that

Reply:
and for that, i believe we did it in order to handle showing the verification once you click on the Buy/Sell page from Advertiser's page since we have to show the verification screen if the user is still not verified. We could handle it from the stores, but in this case, since its only done here, I just used the search params to handle it easily

Reply:
so let's say if the user is not verified yet, but they change the param manually to true. in this case they will be able to see private pages which they are not supposed to see. Are we also checking if the user is verified in those pages/components?

Reply:
hmmm not really, it just means they can just see the buy/sell screen again, but if they click on Buy/Sell USD button, itll still show the verification screen.

Even if they manually change it, it won't reveal anything, its just to handle showing the Verification screen in buy/sell page if we were redirected from Advertiser's page

Reply:
ok. thanks @ameerul-deriv